packages/engine/lib/memory/src/arrow/ipc/serialize/LICENSE

```diff
@@ -0,0 +1,2 @@
+This code is licensed under the Apache License (v2.0). A copy of the license may be found at
+https://github.com/jorgecarleitao/arrow2/blob/main/LICENSE
```

Comment:
Could you add a comment to the functions copied in this directory and state what was added/changed? Similar to our old IPC module.

Reply:
Awesome, thank you very much!